### ISSUE TITLE
NTR: Muscle biopsy: sulfide:quinone oxidoreductase activity low (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9446]
New term request Muscle biopsy: sulfide:quinone oxidoreductase activity low (elthat_211130022818): SQOR  UMLS: 5543168

**Comment by pnrobinson** [comment]
@MickeySegal
I assume you are referring to https://omim.org/entry/619221
There does not seem to be a need to have both "liver" and "muscle" versions of this term, so I would propose merging this with  https://github.com/obophenotype/human-phenotype-ontology/issues/9445

**Comment by MickeySegal** [comment]
Agree.  I will tag that so I add the new code to both our findings.

**Comment by pnrobinson** [comment]
@MickeySegal 
added as
HP:0034555
Reduced sulfide:quinone oxidoreductase activity
def: Reduced activity of the enzyme sulfide:quinone oxidoreductase, which typically is measured in a biopsy specimen from liver or muscle.
